adhocracy4/actions/verbs.py

```diff
@@ -0,0 +1,12 @@
+# http://activitystrea.ms/registry/verbs/
```

Comment:
This whole modules could be replaced by an python enum. Which is simply didn't knew when I implemented this.

    Verbs(enum.Enum):
      CREATE = 'create'
      ADD = 'add'
      UPDATE = 'updated'
      COMPLETE = 'complete'

I am not sure if that really is an improvement, since the `Enum` is rather new (introduced in Python 3.4). There is no proper Django Support (like an Enum field).